assignments/objects.js

```diff
@@ -18,27 +18,67 @@ let example = {
 }
 
 // Write your intern objects here:
+let one = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
 
+let two = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function () {
+    console.log("Hello, my name is " + this.name + "!")
+  }
+}
 
+let three = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M"
+}
+
+let four = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M"
+}
+
+let five = {
+  id: 5,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function (num1, num2) {
+    return num1 * num2
+  }
+}
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(one.name)
 // Kennan's ID
-
+console.log(two.id)
 // Keven's email
-
+console.log(three.email)
 // Gannie's name
-
+console.log(four.name)
 // Antonietta's Gender
-
+console.log(five.gender)
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+two.speak()
```

Comment:
👍 

You can also add methods to objects outside of the initial declaration. So you could do:

```
two.speak = function() {
    console.log("Hello, my name is Kennan!");
}
```

and it would add the `.speak()` method to the object.